libenv/unix_iface.c

```diff
@@ -849,6 +849,11 @@ static void InitIgnoreInterfaces()
 
     if ((fin = fopen(filename, "r")) == NULL)
     {
+        Log(LOG_LEVEL_VERBOSE,
+            "Failed to open interface exception file %s: %s",
+            filename,
+            GetErrorStr());
```

Comment:
Ideally only `ENOENT` should be a verbose message, all the other errors should be error messages. But not a biggie.